pkg/controller/machine.go

```diff
@@ -562,6 +562,16 @@ func (c *controller) machineDelete(machine *v1alpha1.Machine, driver driver.Driv
 			}
 
 			err = driver.Delete()
+
```

Comment:
The error for the above call is not there anymore because of this change.

Reply:
Good catch, will fix it. Thanks.